src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,183 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+Commands to manage Azure container registries
+-------------
+::
+
+    Group
+        az acr: Commands to manage Azure container registries.
+
+    Subgroups:
+        credential: Manage admin user credential for Azure container registries.
+        repository: Manage repositories for Azure container registries.
+        storage   : Manage storage accounts for Azure container registries.
+
+    Commands:
+        create    : Create a container registry.
+        delete    : Delete a container registry.
+        list      : List container registries.
+        show      : Get a container registry.
+        update    : Update a container registry.
+
+Create a container registry
+-------------
+::
+
+    Command
+        az acr create: Create a container registry.
+
+    Arguments
+        --location -l       [Required]: Location.
+        --name -n           [Required]: Name of container registry.
+        --resource-group -g [Required]: Name of resource group.
+        --app-id                      : The app id of an existing service principal. If provided, no
+                                        --new-sp or -p should be specified.
+        --enable-admin                : Enable admin user.
+        --new-sp                      : Create a new service principal. If provided, no --app-id should
```

Comment:
The create_for_rbac PR is still pending, we will remove the code when it is merged.